Pipfile

```diff
@@ -0,0 +1,34 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+aniso8601 = "==9.0.0"
+attrs = "==20.3.0"
+certifi = "==2020.12.5"
+chardet = "==4.0.0"
+click = "==7.1.2"
+Flask = "==1.1.2"
+Flask-Caching = "==1.9.0"
+Flask-RESTful = "==0.3.8"
+idna = "==2.10"
+iniconfig = "==1.1.1"
+itsdangerous = "==1.1.0"
+Jinja2 = "==2.11.3"
+MarkupSafe = "==1.1.1"
+pluggy = "==0.13.1"
+py = "==1.10.0"
+pyparsing = "==2.4.7"
+pytest = "==6.2.2"
+pytz = "==2021.1"
+requests = "==2.25.1"
+six = "==1.15.0"
+toml = "==0.10.2"
+urllib3 = "==1.26.3"
+Werkzeug = "==1.0.1"
```

Comment:
It is important to not let your dependency versions unbounded, but pinning them completely also prevent error fixes that were made upstream to reach your application, as well as performance improvements. 

Fortunately, though, we can rely on [semantic versioning](https://semver.org/) to ensure that only backward compatible changes can implicitly come downstream. In fact, pipenv's main documentation has a good recommendation about this.